dynamiq/utils/feedback.py

```diff
@@ -0,0 +1,35 @@
+from dynamiq.callbacks.streaming import AsyncStreamingIteratorCallbackHandler, StreamingEventMessage
+from dynamiq.runnables import RunnableConfig
+from dynamiq.types.feedback import FeedbackMethod
+
+
+def send_message(
+    event_message: StreamingEventMessage, config: RunnableConfig, approval_type: FeedbackMethod = FeedbackMethod.STREAM
+) -> None:
+    """Emits message to the queue of AsyncStreamingIteratorCallbackHandler
```

Comment:
I don't see any restrictions that it support just by `AsyncStreamingIteratorCallbackHandler`